reporter/s3.go

```diff
@@ -103,34 +99,41 @@ func (w S3Writer) Write(rs ...models.ScanResult) (err error) {
 	return nil
 }
 
+// ErrBucketCheckExist : bucket cannot check the existence because s3:ListBucket or s3:ListAllMyBuckets is not allowed
+var ErrBucketCheckExist = xerrors.New("bucket cannot check the existence because s3:ListBucket or s3:ListAllMyBuckets is not allowed")
+
 // Validate check the existence of S3 bucket
 func (w S3Writer) Validate() error {
 	svc, err := w.getS3()
 	if err != nil {
-		return err
+		return xerrors.Errorf("Failed to get s3 client. err: %w", err)
 	}
 
-	result, err := svc.ListBuckets(&s3.ListBucketsInput{})
-	if err != nil {
-		return xerrors.Errorf("Failed to list buckets. err: %w, profile: %s, region: %s",
-			err, w.Profile, w.Region)
+	// s3:ListBucket
+	_, err = svc.HeadBucket(context.TODO(), &s3.HeadBucketInput{Bucket: aws.String(w.S3Bucket)})
```

Comment:
Just my guess, context.TODO() is used In the example codes because it does not know which context is appropriate.
In this case, context.background() is more appropriate here? (although the implementations are the same)

Reply:
When using `context.background()`, I think it is necessary to bring the context from above, such as by taking it as a function argument.
For temporary context creation, TODO seems to be preferable as before.

Reply:
Thanks! Understood!!